client/src/components/table/Table.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import { AgGridReact } from 'ag-grid-react';
+
+import 'ag-grid-community/dist/styles/ag-grid.css';
+import 'ag-grid-community/dist/styles/ag-theme-alpine.css';
+
+const Table = () => {
+  const [columnDefs] = useState([
+    { headerName: 'Name', field: 'name', width: 100, sortable: true },
+    { headerName: 'Email', field: 'email', width: 180, sortable: true },
+    { headerName: 'Phone #', field: 'phone', width: 160, sortable: true },
+    { headerName: 'NetID', field: 'netid', width: 120, sortable: true },
+    { headerName: 'UIN', field: 'uin', width: 120, sortable: true },
+    { headerName: 'Major', field: 'major', width: 120, sortable: true },
+    {
+      headerName: 'Birth Date',
+      field: 'birthdate',
+      width: 120,
+      sortable: true,
+    },
+    { headerName: 'Github', field: 'github', width: 140, sortable: true },
+    { headerName: 'Snapchat', field: 'snapchat', width: 140, sortable: true },
+    { headerName: 'Instagram', field: 'instagram', width: 140, sortable: true },
+    { headerName: 'Grad Sem/Yr', field: 'grad', width: 190, sortable: true },
+    {
+      headerName: 'Generation',
+      field: 'generation',
+      width: 120,
+      sortable: true,
+    },
+    { headerName: 'Location', field: 'location', width: 120, sortable: true },
+    { headerName: 'Role', field: 'role', width: 160, sortable: true },
+    { headerName: 'Level', field: 'level', width: 120, sortable: true },
+    { headerName: 'Status', field: 'status', width: 120, sortable: true },
+    { headerName: 'Dues', field: 'dues', width: 100, sortable: true },
+    {
+      headerName: 'Address Form',
+      field: 'addressform',
+      width: 140,
+      sortable: true,
+    },
+    {
+      headerName: 'Next Semester',
+      field: 'semesterform',
+      width: 160,
+      sortable: true,
+    },
+  ]);
+  const [rowData] = useState([
+    {
+      name: 'Amit',
+      email: 'sawhney4@illinois.edu',
+      phone: '(309) 838-5466',
+      netid: 'sawhney4',
+      uin: 659426199,
+      major: 'CS + Math',
+      birthdate: '02/14/2002',
+      github: 'amit-sawhney',
+      snapchat: 'amit_sawhney02',
+      instagram: 'amit__02',
+      grad: 'Spring 2024 (Freshman)',
+      generation: 'Fall 2020',
+      location: 'On-campus',
+      role: 'Software Developer',
+      level: 'Member',
+      status: 'Active',
+      dues: 'Y',
+      addressform: 'Y',
+      semesterform: 'Y',
+    },
+  ]);
+  return (
+    <div
+      className="ag-theme-alpine"
+      style={{ height: 400, width: '100%', margin: 'auto' }}
```

Comment:
Could you move this to a seperate css file?

Reply:
Yes, I was planning on doing that. The only reason I didn't was because of the template-based approach I mentioned above. I had no intentions of actually developing this table at all in this PR, I just wanted to make it roughly render. I know there are built in classes as you can see from the top of this file that we may be using, so I didn't want to get too carried away with making style sheets for this.